flexdll.c

```diff
@@ -561,9 +579,20 @@ void flexdll_dlclose(void *u) {
 
 
 void *flexdll_dlsym(void *u, const char *name) {
-  if (u == &main_unit) return find_symbol_global(NULL,name);
-  else if (NULL == u) return find_symbol(&static_symtable,name);
-  else return find_symbol(((dlunit*)u)->symtbl,name);
+  void *res;
+  err_t * err;
+  err = get_tls_error(TLS_ERROR_NOP);
+  if (err == NULL) return NULL;
+
+  if (WaitForSingleObject(units_mutex, INFINITE) == WAIT_OBJECT_0) {
```

Comment:
```suggestion
  if (WaitForSingleObject(units_mutex, INFINITE) != WAIT_OBJECT_0) {
```
ouch, sorry!

Reply:
No problem! I didn't notice in the CI - but found out locally when tests weren't passing :sweat_smile: